app/components/Header/DropdownContainer/FadeContents.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import React, { forwardRef } from 'react';
+import type { Node } from 'react';
+import styled, { keyframes } from 'styled-components';
```

Comment:
Why is there styled-components in here?

Reply:
It made life a whole lot easier, but I plan on removing it in the end, it's just a low priority right now